plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -107,6 +107,14 @@ func (p *plugin) validateArgs() (err error) {
 		// the additional values filepaths must be relative to the kust root
 		p.AdditionalValuesFiles[i] = filepath.Join(p.h.Loader().Root(), file)
 	}
+	for key, file := range p.AdditionalValuesFromFileContent {
```

Comment:
You need to run a make target to update the mirror code under `api/internal/plugins` as well. I think it's `generate-plugins` or something similar, you can peruse the make targets to find it. Additionally, this feature requires tests, both under `api/krusty` and in this directory.

Reply:
@natasha41575 thanks for your feedback. I added tests and the generated mirror code.